src/shared/components/ThreeDotsIcon/ThreeDotsIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import classes from "./ThreeDotsIcon.module.scss";
+
+const ThreeDotsIcon: React.FC = () => (
```

Comment:
i think there is no need for this icon, you can easily do it with css, but it's just a recommendation